node/src/components/contract_runtime/rewards/tests.rs

```diff
@@ -0,0 +1,861 @@
+use crate::testing::{map, set};
+use casper_types::{
+    testing::TestRng, AsymmetricType as _, EraId, RewardedSignatures, TestBlockBuilder,
+};
+use once_cell::sync::Lazy;
+use std::{iter, ops::Deref};
+
+use self::constructors::RewardsInfoConstructor;
+
+use super::*;
+use convert::ratio;
+
+fn val(n: u8) -> PublicKey {
+    let mut buf = [0; 32];
+    (0..22).for_each(|i| buf[i] = n);
```

Comment:
Is it `22` intentionally, or a typo of `32`?

Reply:
It is, it's a value for which all the buffers are valid public keys.